src/components/Board/index.spec.js

```diff
@@ -109,11 +109,12 @@ describe('<Board />', () => {
             })
           })
 
-          it('calls the onCardDragEnd callback passing the card move coordinates', () => {
+          it('calls the onCardDragEnd callback passing the card and move coordinates', () => {
             expect(onCardDragEnd).toHaveBeenCalledTimes(1)
             expect(onCardDragEnd).toHaveBeenCalledWith(
               { fromPosition: 0, fromLaneId: 1 },
-              { toPosition: 1, toLaneId: 1 }
+              { toPosition: 1, toLaneId: 1 },
+              { id: 1, description: 'Card content', title: 'Card title 1' }
```

Comment:
Since the uncontrolled board has the board as the first argument and to keep this pattern we should pass the card as the second argument thus making a breaking change, I think we should pass the card as the first argument here.